e2e/process-services/widgets/attach-folder-widget.e2e.ts

```diff
@@ -62,8 +64,9 @@ describe('Attach Folder widget', () => {
    });
 
     beforeEach(async () => {
-        const urlToNavigateTo = `${browser.baseUrl}/activiti/apps/${deployedApp.id}/tasks/`;
-        await BrowserActions.getUrl(urlToNavigateTo);
+        await navigationBarPage.clickHomeButton();
```

Comment:
I've seen others that don't need to go to the home, do we need it?

Reply:
yep we need to refresh the form

Reply:
maybe I can try to click only refresh form let me try